index.ts

```diff
@@ -340,10 +348,14 @@ export class PullFlatList<T> extends Component<PullFlatListProps<T>, State<T>> {
     }
   }
 
-  public stopPrefixListener() {
+  public stopPrefixListener(cb?: () => void) {
     if (this.prefixReadable) {
       this.prefixReadable(true, () => {});
     }
+
+    if (this.unmounting) {
+      cb?.();
+    }
```

Comment:
should there be an `else` clause that updates the component state here, similar to what's done in `stopScrollListener`?

Reply:
I don't think we need to :)